v0/destinations/active_campaign/transform.js

```diff
@@ -529,11 +501,8 @@ const screenRequestHandler = async (message, category, destination) => {
   payload.actid = destination.Config.actid;
   payload.key = destination.Config.eventKey;
   payload.visit = encodeURIComponent(
-    `{email : ${
-      message.context.traits.email
-        ? message.context.traits.email
-        : message.context.traits.traits.email
-    }}`
+    `{"email":"${get(message, "context.traits.email") ||
+      get(message, "context.traits.traits.email")}"}`
```

Comment:
What happens if there is not email in context.traits or context.traits.traits? If AC doesn't through any error it is ok.

Reply:
It doesn't throw any error.